update_package.sh

```diff
@@ -34,6 +34,9 @@ sed -i 's/\[\*\*AnyType\*\*\](AnyType\.md)/AnyType/g' docs/DefaultApi.md
 sed -i 's/\\link{AnyType}/AnyType/g' R/default_api.R
 sed -i 's#${module}.\${model}#module + "." + model#g' R/default_api.R
 
+find . -type f -exec sed -i 's/\(InstancesInstanceId\|StoragesS3BucketNamePermissions\)[A-Za-z]*\(Get\|Post\|Put\|Delete\|Patch\)/\2/g' {} \;
+find . -type f -exec sed -i 's/CreateInstance/CreateInstance/g' {} \;
```

Comment:
This line looks like it doesn't do anything, but maybe I'm missing something obvious?

Reply:
Oh, the first part should say something like CreateInstanceInstances

```suggestion
find . -type f -exec sed -i 's/CreateInstancesInstance/CreateInstance/g' {} \;
```

I think my command changed itself 😅